### PR TITLE
Get rif of prefixed `requestAnimationFrame` leftovers

### PR DESCRIPTION
**Pull request by mourner**

### CODE REVIEW

**Comment by mourner** [comment]
Well, this is pretty weird — I don't know why this would fail CI so much.

**Comment by mourner** [comment]
This seems to work https://github.com/Leaflet/Leaflet/tree/cleanup-raf-leftovers-2 — really can't comprehend what's wrong with the current changes though.

**Comment by jonkoops** [comment]
@mourner I had this exact same issue when I tried to clean up the RAF code. It seems to have something to do with how the test suite handles the RAF functions, as it does actually function in manual testing.

File: src/core/Util.js [hunk 1]
```diff
@@ -156,14 +156,8 @@ export function template(str, data) {
...
-// inspired by https://paulirish.com/2011/requestanimationframe-for-smart-animating/
-
-function getPrefixed(name) {
-	return window[`webkit${name}`] || window[`moz${name}`] || window[`ms${name}`];
-}
-
-export const requestFn = window.requestAnimationFrame || getPrefixed('RequestAnimationFrame');
-export const cancelFn = window.cancelAnimationFrame || getPrefixed('CancelAnimationFrame') || getPrefixed('CancelRequestAnimationFrame');
+const requestFn = window.requestAnimationFrame;
+const cancelFn = window.cancelAnimationFrame;
```
[the remark this answers is not in the document]
Comment (mourner) [reply src/core/Util.js:160]: @Falke-Design see the comments above — I removed them initially but it fails all tests on CI for reasons I can't understand.

**Comment by mourner** [comment]
@jonkoops put the variables back and it works (no idea why though) — ready for review now.

File: src/core/Util.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (IvanSanchez) [reply src/core/Util.js:160]: I'm slightly concerned about removing the `export` from those. Don't know if there are any plugins that depend on `L.Util.requestFn`.

[the remark this answers is not in the document]
Comment (Falke-Design) [reply src/core/Util.js:160]: > @Falke-Design see the comments above — I removed them initially but it fails all tests on CI for reasons I can't understand.

Sorry didn't know that you removed them, was not visible in your commit history. 👍

[the remark this answers is not in the document]
Comment (Falke-Design) [reply src/core/Util.js:160]: > I'm slightly concerned about removing the export from those. Don't know if there are any plugins that depend on L.Util.requestFn.

Or we say it is a cleanup of the legacy code like all the other DOMUtils that we removed for `v2`. But I'm also fine with exporting it

[the remark this answers is not in the document]
Comment (IvanSanchez) [reply src/core/Util.js:160]: Removing them for v2 should be fine.

**Review by Falke-Design — APPROVED** [review]

File: src/core/Util.js (hunk 1 above)
Comment (jonkoops) [inline src/core/Util.js:160]: Perhaps we should be using [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) for newer code? 

```suggestion
const requestFn = globalThis.requestAnimationFrame;
const cancelFn = globalThis.cancelAnimationFrame;
```

[the remark this answers is not in the document]
Comment (jonkoops) [reply src/core/Util.js:160]: Yes, these should be removed for v2 :+1:

File: src/core/Util.js [hunk 2]
```diff
@@ -176,7 +170,5 @@ export function requestAnimFrame(fn, context) {
 // @function cancelAnimFrame(id: Number): undefined
 // Cancels a previous `requestAnimFrame`. See also [window.cancelAnimationFrame](https://developer.mozilla.org/docs/Web/API/window/cancelAnimationFrame).
 export function cancelAnimFrame(id) {
-	if (id) {
```
Comment (jonkoops) [inline src/core/Util.js:179]: Any particular reason we are no longer checking the `id` here?

**Review by jonkoops — APPROVED** [review]
LGTM, just some small comments.

File: src/core/Util.js (hunk 2 above)
Comment (mourner) [reply src/core/Util.js:179]: It seems redundant — e.g. `cancelAnimationFrame(null)` is still a noop, and `requestAnimationFrame` per the spec returns a non-zero ID value.

**Pull request merged**